### ISSUE TITLE
Not able to push my local changes

### ISSUE THREAD

**Issue by Khushi-1412** [incubateind/gdtc-Day2#48]
while pushing it is showing
remote: Permission to incubateind/gdtc-Day2.git denied to Khushi-1412.
fatal: unable to access 'https://github.com/incubateind/gdtc-Day2.git/': The requested URL returned error: 403

**Comment by rexyagami** [comment]
@Khushi-1412 hello, have you forked the repository?

**Comment by Khushi-1412** [comment]
Yes I have forked. Sent from Mail for Windows 10 From: Rex YagamiSent: 07 February 2021 18:21To: incubateind/gdtc-Day2Cc: Khushi Kumari; MentionSubject: Re: [incubateind/gdtc-Day2] Not able to push my local changes (#48) @Khushi-1412 hello, have you forked the repository?—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by rexyagami** [comment]
@Khushi-1412 try pushing on the changes on the forked repo on your profile
